### PR TITLE
Fix filter issue for vapb and kyverno

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Ref: 

- https://issues.redhat.com/browse/ACM-15839
- https://issues.redhat.com/browse/ACM-15836
- https://issues.redhat.com/browse/ACM-15834

### CODE REVIEW

**Review by JustinKuli — APPROVED** [review]
LGTM

File: frontend/src/routes/Governance/discovered/ByCluster/common.tsx [hunk 1]
```diff
@@ -108,6 +108,7 @@ export const byClusterCols = (
   subscriptions: Subscription[],
   channels: Channel[],
   policyKind: string,
+  diabledSeverityTooltip: boolean,
```
Comment (mprahl) [inline frontend/src/routes/Governance/discovered/ByCluster/common.tsx:111]: It'd be nice to fix the typo in the variable name eventually `diabledSeverityTooltip` => `disabledSeverityTooltip`

File: frontend/src/routes/Governance/discovered/DiscoveredPolicies.test.tsx [hunk 2]
```diff
@@ -569,5 +569,13 @@ describe('useFetchPolicies custom hook', () => {
         name: /require-owner-labels Kyverno ClusterPolicy Audit Medium 1 Local/,
       })
     ).toBeInTheDocument()
+
+    // Validate filter
+    await waitForText('Filter')
+
+    screen.getByRole('button', { name: 'Options menu' }).click()
+    screen.logTestingPlaygroundURL()
```
Comment (mprahl) [inline frontend/src/routes/Governance/discovered/DiscoveredPolicies.test.tsx:577]: This log should have been removed

Comment (yiraeChristineKim) [reply frontend/src/routes/Governance/discovered/DiscoveredPolicies.test.tsx:577]: Gooood Catch Thanks!! let me open a PR

**Pull request merged**